projects/plugins/jetpack/modules/shortcodes/vimeo.php

```diff
@@ -246,53 +246,6 @@ class_exists( 'Jetpack_AMP_Support' )
 }
 add_shortcode( 'vimeo', 'vimeo_shortcode' );
 
-/**
- * Callback to modify output of embedded Vimeo video using Jetpack's shortcode.
- *
- * @since 3.9
- *
- * @param array $matches Regex partial matches against the URL passed.
- * @param array $attr    Attributes received in embed response.
- * @param array $url     Requested URL to be embedded.
- *
- * @return string Return output of Vimeo shortcode with the proper markup.
- */
-function wpcom_vimeo_embed_url( $matches, $attr, $url ) {
-	$vimeo_info = array( $url );
-
-	// If we are able to extract a video ID, use it in the shortcode instead of the full URL.
-	if ( ! empty( $matches['video_id'] ) ) {
-		$vimeo_info = array( 'id' => $matches['video_id'] );
-	}
-
-	return vimeo_shortcode( $vimeo_info );
-}
-
-/**
- * For bare URLs on their own line of the form.
- *
- * Accepted formats:
- * https://vimeo.com/289091934/cd1f466bcc
```

Comment:
It would be worth making sure that Core handles this format as well. Those are private videos, iirc. See this issue for more information:
https://github.com/Automattic/wp-calypso/issues/57257

Reply:
Will check! I thought it's not supported by Vimeo now 😂

Reply:
> An example of a live video would be https://vimeo.com/635643256/b6a59a4d1e
> An example of a video to purchase would be https://vimeo.com/555942796/e712a2e45d

I tested the video that needs to purchase, and it works well. However, the above live video is outdated so I cannot check whether the live video with the above format works or not. But I tested another live video, https://vimeo.com/1003283038, and it works!

In short, I assume Core supports the vimeo video with the above format, e.g.: https://vimeo.com/555942796/e712a2e45d

Reply:
I think it supports all format as the Core uses `http://vimeo.com/api/oembed.json` endpoint to ask the information about the url, e.g. `http://vimeo.com/api/oembed.json?url=http://vimeo.com/76979871`

See http://developer.vimeo.com/apis/oembed